src/3D/setup3d.F90

```diff
@@ -799,6 +807,13 @@ subroutine setup(fileName, fileType)
             end do
         end do patchLoop
 
+        ! if there was an error in the BC setup stop
+        if (bcError) then
+            print *, 'pyHyp exited due to one or more issues with mesh boundary conditions. &
+&            See above error printouts.'
```

Comment:
The leading `&` with all the spaces will create a large gap in the string. For the line continuation to have consistent spacing, place the `&` right before `See`, i.e., `&See`.